src/hrtf/neighbors.c

```diff
@@ -108,21 +108,21 @@ MYSOFA_EXPORT struct MYSOFA_NEIGHBORHOOD *mysofa_neighborhood_init(struct MYSOFA
 				neighbor->index[i * 6 + 4] = index;
 				break;
 			}
-			radius *= 1.5;
```

Comment:
Why adding, not multipling?

Reply:
1) to be consistant with how theta and phi are incremented, 2) because the data I'm using have been simulated/measured on regular (understand linear) radius grids (e.g. grid ILD coefficients for nearby sources). 
The multiplying you're using here results in an exponential increase. I can see how it speeds up the lookup table creation and how it "fits" to psychological "less res. required as distance increases". It results in overwriting measurement grid precision with your own arbitrary criteria though.